opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/SingleFieldQuery.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.opensearch.storage.script.filter.lucene.relevance;
+
+import java.util.Map;
+import org.opensearch.index.query.QueryBuilder;
+import org.opensearch.sql.expression.NamedArgumentExpression;
+
+/**
+ * Base class to represent relevance queries that search a single field only.
```

Comment:
I think the description could have a little more context about what it does do, rather than just about what fields it contains.

Reply:
See https://github.com/Bit-Quill/opensearch-project-sql/pull/98/commits/470baf5e5a7f06094db88a38f8b4ac7976cccc73